tests/smb3parse/util/test_code_edit_dict.py

```diff
@@ -99,3 +99,17 @@ def test_write_value_incorrect_length():
 
     edit.write("test_2")
     assert "default" == edit.read()
+
+def test_valid_option():
+    testdict = {
+        "default": default_value,
+        "test_1": bytearray([0x01, 0x01, 0x01]),
+        "test_2": bytearray([0x02, 0x02, 0x02, 0x02]),
+    }
+
+    edit = CodeEditDict(get_test_rom(), 0x100, len(default_value), [], testdict, [])
```

Comment:
CodeEditDict does not accept a list as parameter.  This should be updated to be a bytearray.